Makefile.opensuse

```diff
@@ -1,19 +1,9 @@
 include Makefile
 
+OS_NAME=opensuse
```

Comment:
We're going to have to do some extra work in this file if we don't want to change the global values.
I realized just today that setting TAG=opensuse-latest will *not match* what @dmulder originally added.
Instead of using the inherited TAG just use "latest" or use an alternate variables for the <X>_REPO_NAME variables.  The local names can continue to use TAG specifically because they're local.  I'll note that the opensuse build does not include the nightly variants so it simplifies things slightly.